urllib3/_backends/__init__.py

```diff
@@ -1,9 +0,0 @@
-from urllib3.packages import six
```

Comment:
I don't really see the point in this file any more, so I truncated it. I guess it would make sense to put `load_backend` in here, and have that be the only interface to this module, but that feels like making a decision. 

Additionally, I'm not sure why a private module (`_backends`) would expose a public interface (via `__all__`)?

Reply:
If I did that then probably my logic was, this is a private API from the user point of view (they shouldn't mention `_backend` at all), and an opaque module from the point of view of the rest of the internal code (they can use the stuff in `_backend.__all__`, but shouldn't go digging deeper).

This is a pretty fine distinction and not particularly important.